
syntax = "proto3";

package microservice;

service Microservice {
	rpc ListNews(Numero) returns (stream News) {
	}	
}

message Numero {
	int32 numero = 1;
}

message News  {
	string id = 1;
	string title = 2;
	string url = 3;
	string publisher = 4;
	string category = 5;
	string story = 6;
	string hostname = 7;
	string time_stamp = 8;
	int32 numero_accessos = 9;
}